app/models/parcel.rb

```diff
@@ -25,6 +27,20 @@ class Parcel < ActiveRecord::Base
 
   scope :newest_first, -> { order(created_at: :desc) }
 
+  def accept!(author=nil)
+    self.update(acceptance_status: 'accepted')
+    operation = Operation.new(parcel: self, kind: 'order_accepted')
+    operation.user = author if author
+    operation.save!
+  end
+
+  def reject!(author=nil)
+    self.update(acceptance_status: 'rejected')
+    operation = Operation.new(parcel: self, kind: 'order_rejected')
+    operation.user = author if author
+    operation.save!
+  end
+
```

Comment:
Jest pytanie czy w ogóle nie byłoby lepiej zrobić tego tak, że jest jedna operacja i to ona decyduje czy robi accept, czy reject - popatrz na dwie powyższe metody i poszukaj części wspólnych.
Poza tym tutaj właśnie użyteczny może być serwis. Przyjmuje `parcel`, przyjmuje `status` i mieli swoje.